tests/test_project/tests/test_admin.py

```diff
@@ -2,8 +2,8 @@
 from django.test import TestCase
 from django.urls import reverse
 
-from . import CreateMixin
 from ..models import Operator, Project, RadiusAccounting
+from . import CreateMixin
```

Comment:
Is there any particular reason for this change?

Reply:
There was an error in build-
`ERROR: /home/travis/build/Weirdo914/openwisp-utils/tests/test_project/tests/test_admin.py Imports are incorrectly sorted.`